docs/developer-docs/daos/nns/proposal-overview.mdx

```diff
@@ -0,0 +1,19 @@
+---
+keywords: [intermediate, governance, concept, nns]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Overview
+
+<MarkdownChipRow labels={["Intermediate", "Governance", "Concept"]} />
+
+A proposal is a suggestion for a change to the Internet Computer protocol.
+As the NNS is an open governance system where anyone can become a participant.
+NNS DAO participants can submit proposals an thereby make suggestions on how the ICP should evolve and they can vote on proposals that other submitted.
+Based on the majority decision, a proposal is then adopted or rejected.
+If a proposal is adopted, it is automatically executed fully on-chain, without the need for manual intervention.
+
+More technically, to participate in governance users can stake some ICP in a neuron (at least 1 ICP are required to create a neuron).
+If a neuron has more than 6 months dissolve delay, it is eligible to participate in voting.
+This means that it can both submit and vote on proposals.
```

Comment:
```suggestion
A proposal is a suggestion for a change to the Internet Computer protocol. The NNS is an open governance system where anyone can become a participant. NNS DAO participants can submit proposals and thereby make suggestions on how the ICP should evolve and can vote on proposals that others have submitted.

Based on the majority decision, a proposal is then adopted or rejected. If a proposal is adopted, it is automatically executed fully on-chain, without the need for manual intervention.

More technically, to participate in the NNS governance, users can stake some ICP tokens in a neuron. At least 1 ICP are required to create a neuron.

If a neuron has more than 6 months dissolve delay, it is eligible to participate in voting. This means that it can both submit and vote on proposals.
```